schemas/com.snowplowanalytics.snowplow/client_session/jsonschema/1-1-0

```diff
@@ -0,0 +1,70 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a client-generated user session",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "client_session",
+		"format": "jsonschema",
+		"version": "1-1-0"
+	},
+	"type": "object",
+	"properties": {
+		"userId": {
+			"type": "string",
+			"pattern": "^[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$|^[0-9a-f]{16}$",
```

Comment:
I wonder if there is any advantage using this pattern rather than the _"format": "uuid"_.
I would use _format uuid_ where possible. IMO it shouldn't be a breaking change for the loader.

Reply:
I don't see a reason why not, changed it to "format": "uuid"!